Chapter 2/1 - Colors/Window.cs

```diff
@@ -0,0 +1,260 @@
+﻿using System;
+using OpenTK;
+using OpenTK.Graphics;
+using OpenTK.Graphics.OpenGL4;
+using OpenTK.Input;
+using LearnOpenTK.Common;
+
+namespace LearnOpenTK
+{
+    //In this chapter we will focus on how to use lighting to make our games and other applications more lifelike
+
+    //In this first part the focus will mainly be on setting up a scene for testing the different coloring options.
+    //We draw two cubes one at 0,0,0 for testing our light shader, the second one is drawn where we have the light.
+    //Furthermore in the shaders we have set up some basic physically based coloring.
+    public class Window : GameWindow
+    {
+        //The vertices are now used to draw cubes, notice how we dont use the texture coords anymore,
```

Comment:
Rephrase a little bit. Perhaps something more along the lines of "For this example, we aren't using texture coordinates. You can use textures with lighting, but for simplicity's sake, we'll just use a solid color here."

Reply:
Fixed in next commit